selftests/deployment/roles/tests/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
     - avocado_vt|default(false)|bool == true
 
 - name: Avocado-VT test 3/5 - dry-run execution
-  shell: avocado run --dry-run -- boot
+  shell: "{{ temporary_dir.path | default('/usr') }}/bin/avocado run --dry-run -- boot"
```

Comment:
We could test if the binary are properly installed into the PATH, so I'm not sure if this is needed. If it is being needed maybe we are missing some steps on the installation process (i.e. activate the virtual env)

Reply:
> We could test if the binary are properly installed into the PATH, so I'm not sure if this is needed. If it is being needed maybe we are missing some steps on the installation process (i.e. activate the virtual env)

Then we need to change all the steps. This fix was to make this step to work like the others.

Reply:
@beraldoleal I looked at this and I haven't found a way to remove that.